### PR TITLE
Add vt

### PR DESCRIPTION
**Pull request by cagaser**
### Checklist
- [x] Pull request details were added to CHANGELOG.md

### CODE REVIEW

File: vt.wdl [hunk 1]
```diff
@@ -0,0 +1,60 @@
+version 1.0
+
+# Copyright (c) 2019 Sequencing Analysis Support Core - Leiden University Medical Center
```
Comment (DavyCats) [inline vt.wdl]: The copyright year seems off here

Comment (cagaser) [reply vt.wdl]: Fixed

Comment (DavyCats) [reply vt.wdl]: It hasn't changed?

Comment (cagaser) [reply vt.wdl]: Ah I was in the wrong branch :smile:

Comment (cagaser) [reply vt.wdl]: Now fixed

**Pull request merged**